File: question18.java
import java.util.Scanner;
public class question18 {
    public static void main(String[] args) {
        // Fibonacci serisinin ilk 10 terimini ekrana basan kod.

        Scanner scanner = new Scanner(System.in);

        int sayi1=1;
        int sayi2=1;

        System.out.println(sayi1);
        System.out.println(sayi2);

        for(int i=0;i<8;i++){
            int temp=sayi1+sayi2;
            sayi1=sayi2;
            sayi2=temp;
            System.out.println(temp);
        }

        scanner.close();
    }
}

File: question18.py
# Fibonacci serisinin ilk 10 terimini yazdiran kod

sayi1=1
sayi2=1

print(sayi1)
print(sayi2)

for i in range(0,8):
    temp=sayi1+sayi2
    sayi1=sayi2
    sayi2=temp
    print(temp)